e2e-tests/gef/cypress/integration/review-ukef-approval-with-issued-and-unissued-facilities-MIA.spec.js

```diff
@@ -0,0 +1,468 @@
+      applicationDetails.exporterSummaryListRowAction(0, 3).should('have.value', '');
+      applicationDetails.exporterSummaryListRowAction(0, 4).should('have.value', '');
+      applicationDetails.exporterSummaryListRowAction(0, 5).should('have.value', '');
+      applicationDetails.exporterSummaryListRowAction(0, 6).should('have.value', '');
+      applicationDetails.exporterSummaryListRowAction(0, 7).should('have.value', '');
+
+      // should not be able to edit eligibility criteria
+      applicationDetails.automaticCoverSummaryListRowAction(0, 0).should('not.exist');
+
+      // abandon link should not exist
+      applicationDetails.abandonLink().should('not.exist');
+      // should not be able to edit ref name
+      applicationDetails.editRefNameLink().should('not.exist');
+      // should not be able to edit supporting info
+      applicationDetails.supportingInfoListRowAction(0, 0).should('not.exist');
+      applicationDetails.supportingInfoListRowAction(0, 1).should('not.exist');
+    });
+
+    it('should be able to submit to checker after making changes', () => {
+      applicationDetails.submitButton().click();
+      applicationSubmission.submitButton().click();
+      applicationSubmission.confirmationPanelTitleFacilities().contains('Issued facilities submitted for checking at your bank');
+    });
+  });
+});
+
+/**
+ * log in as checker
+ * ensure everything locked
+ * submit to UKEF
+ * ensure correct success message and text are shown
+*/
+context('Submit to UKEF', () => {
+  describe('Check all fields are populated and return to maker', () => {
+    beforeEach(() => {
+      Cypress.Cookies.preserveOnce('connect.sid');
+      cy.login(CREDENTIALS.CHECKER);
+      cy.visit(relative(`/gef/application-details/${dealId}`));
+    });
+
+    it('should show changed facilities in task comments box with correct heading including newly issued', () => {
+      applicationPreview.updatedUnissuedFacilitiesHeader().contains('The following facility stages have been updated to issued:');
+      applicationPreview.updatedUnissuedFacilitiesList().contains(unissuedFacilitiesArray[0].name);
+    });
+
+    it('should show correct status', () => {
+      statusBanner.bannerStatus().contains(CONSTANTS.DEAL_STATUS.BANK_CHECK);
+    });
+
+    it('submit to ukef and return to maker buttons exist and able to return to maker', () => {
+      applicationPreview.submitButton().should('exist');
+      applicationPreview.submitButton().click();
+      applicationSubmission.submitButton().click();
+      applicationSubmission.confirmationPanelTitle().contains(`${toTitleCase(CONSTANTS.DEAL_SUBMISSION_TYPE.MIA)} submitted to UKEF`);
+      // check that correct text is displayed under confirmation panel
+      applicationSubmission.confirmationText().contains('We’ll send you a confirmation email shortly, once we’ve acknowledged your inclusion notice.');
+    });
+  });
+});
```

Comment:
Not the nicest filename but should we add 'resubmit' or something to the filename? i.e
`review-ukef-approval-with-issued-and-unissued-facilities-MIA-and-resubmit-to-ukef.spec.js `

Reply:
done